apps/api/src/queries/buildViewModelForResource/viewModels/playlist.view-model.ts

```diff
@@ -62,6 +70,7 @@ export class PlaylistViewModel extends BaseResourceViewModel {
             allResourceCompositeIdsToFind.some((compositeId) => compositeId.id === id)
         );
 
+        // note that the client has to append the base URL
```

Comment:
Let's test this carefully to be sure we haven't broken the client for playlists.